addons/sale_loyalty_delivery/models/sale_order.py

```diff
@@ -37,6 +37,8 @@ def _get_not_rewarded_order_lines(self):
 
     def _get_reward_values_free_shipping(self, reward, coupon, **kwargs):
         delivery_line = self.order_line.filtered(lambda l: l.is_delivery)
+        if delivery_line:
+            delivery_line = delivery_line[0]
```

Comment:
```suggestion
        delivery_line = self.order_line.filtered(lambda l: l.is_delivery)[:1]
```
?

change seems safe for me, it breaks below if there are multiple lines anyway.

Reply:
Indeed. I knew there were a shorter way, but couldn't find it back.
It also seems safe to me, but wanted to double-check with you.